src/main/kotlin/no/nav/medlemskap/regler/v1/Hovedregler.kt

```diff
@@ -6,69 +6,71 @@ import no.nav.medlemskap.regler.common.*
 import no.nav.medlemskap.regler.common.Svar.NEI
 
 class Hovedregler(datagrunnlag: Datagrunnlag) {
-    private val reglerForEøsStatsborgerskap = ReglerForEøsStatsborgerskap.fraDatagrunnlag(datagrunnlag)
-    private val reglerForNorskStatsborgerskap = ReglerForNorskStatsborgerskap.fraDatagrunnlag(datagrunnlag)
-    private val reglerForRegistrerteOpplysninger = ReglerForRegistrerteOpplysninger.fraDatagrunnlag(datagrunnlag)
+    private val reglerForStatsborgerskap = ReglerForStatsborgerskap.fraDatagrunnlag(datagrunnlag)
+    private val reglerForMedl = ReglerForMedl.fraDatagrunnlag(datagrunnlag)
     private val reglerForNorskeStatsborgere = ReglerForNorskeStatsborgere.fraDatagrunnlag(datagrunnlag)
     private val reglerForEøsBorgere = ReglerForEøsBorgere.fraDatagrunnlag(datagrunnlag)
     private val reglerForAndreStatsborgere = ReglerForAndreStatsborgere.fraDatagrunnlag(datagrunnlag)
     private val reglerForArbeidsforhold = ReglerForArbeidsforhold.fraDatagrunnlag(datagrunnlag)
 
     fun kjørHovedregler(): Resultat {
-        val ytelse = reglerForRegistrerteOpplysninger.ytelse
+        val ytelse = reglerForMedl.ytelse
 
-        val resultatRegistrertOpplysninger = reglerForRegistrerteOpplysninger.kjørRegelflyt()
-        val resultatNorskStatsborgerskap = reglerForNorskStatsborgerskap.kjørRegelflyt()
-        val resultatEøsStatsborgerskap = reglerForEøsStatsborgerskap.kjørRegelflyt()
-        val resultatForStatborgerskap = bestemReglerForStatsborgerskap(resultatEøsStatsborgerskap, resultatNorskStatsborgerskap)
-                .kjørRegelflyt()
-        val resultatForArbeidsforhold = reglerForArbeidsforhold.kjørRegelflyt()
+        val resultater = mutableListOf<Resultat>()
 
-        val resultater = listOf(
-                resultatRegistrertOpplysninger,
-                resultatEøsStatsborgerskap,
-                resultatNorskStatsborgerskap,
-                resultatForArbeidsforhold,
-                resultatForStatborgerskap
-        )
+        resultater.addAll(reglerForMedl.kjørRegelflyter())
+        val resultatStatsborgerskap = reglerForStatsborgerskap.kjørRegelflyter()
```

Comment:
Du "mellomlagrer" denne men ingen av de andre?

Reply:
resultatStatsborgerskap brukes to steder under, det er derfor den mellomlagres.